### DISCUSSION TITLE
Why not add a tag function to every weapon?

### DISCUSSION THREAD

**Discussion by hanaru1** [Nereziel/cs2-WeaponPaints#289] [Ideas]
Hi,Nereziel!It's of my happiness to write this article to you.The plugin WeaponPaints you created is so awsome and I feel delighted when I first try this plugin.However,I think the only shortage of WeaponPaints is that it doesn't have any function to change the tag of every weapon or knife.I know it might not be so necessary but I really expect it,so I'm writing this article to give such advice to you.I would be appreciated if you could take my suggestions into consideration. ^_^

**Comment by Nereziel** [comment]
We plan to add more features.
